index.md

```diff
@@ -39,34 +39,41 @@ Here are a few values we think you will receive from participating in this proje
 
 ## Timeline
 
-- **October 2019**: Opening of the applications
-- **End of November 2019**: Closing of the applications
-- **Mid of December 2019**: Successful applicants announced
-- **January 2020**: Start of the 12-week program
+- **October 25, 2019**: Opening of the applications
+- **December 8, 2019**: Closing of the applications
```

Comment:
I'd suggest end of november (30th) but a week later is fine too.

Reply:
It was to align with Emmy (see her announcement on Slack)

Reply:
oh, nice one! ignore me :D